specification/maps/data-plane/Creator/preview/2022-01-01-preview/features.json

```diff
@@ -0,0 +1,959 @@
+      "x-ms-parameter-location": "method"
+    },
+    "CollectionId": {
+      "name": "collectionId",
+      "in": "path",
+      "type": "string",
+      "required": true,
+      "description": "Identifier (name) of a specific collection",
+      "x-ms-parameter-location": "method"
+    },
+    "FeatureId": {
+      "name": "featureId",
+      "in": "path",
+      "type": "string",
+      "description": "Local identifier of a specific feature",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "FeatureBody": {
+      "in": "body",
+      "name": "feature",
+      "required": true,
+      "description": "A valid `GeoJSON Feature` object type. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.2) for details.",
+      "schema": {
+        "$ref": "#/definitions/Feature"
+      }
+    },
+    "FeatureAttributesBody": {
+      "in": "body",
+      "name": "featureAttributes",
+      "required": true,
+      "description": "The geometry and properties of a feature.",
+      "schema": {
+        "$ref": "#/definitions/FeatureAttributes"
+      }
+    }
+  },
+  "paths": {
+    "/features/datasets/{datasetId}": {
+      "get": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nCreator makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. [This](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps) article introduces concepts and tools that apply to Azure Maps Creator. For more information with regards to ontology, please read this [article](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2) describing the Facility ontology.\n\n The Features API is part of Creator. Features API follows the [Open Geospatial Consortium API standard for Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v2/dataset/create). \nA dataset consists of multiple feature collections. A feature collection is a collection of features of a similar type, based on a common schema.\nThe Get Landing Page API provides links to the API definition, the Conformance statements  and the metadata about the feature data in this dataset.",
+        "operationId": "Features_GetLandingPage",
+        "x-ms-examples": {
+          "GetLandingPage": {
+            "$ref": "./examples/features/Features_GetLandingPage.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../2.0/dataset.json#/parameters/DatasetId"
```

Comment:
References to dataset.json will be updated to latest version pending doc changes.